_posts/2021-04-12-homebrew-3.1.0.md

```diff
@@ -0,0 +1,45 @@
+---
+title: 3.1.0
+author: MikeMcQuaid
+redirect_from: /blog/3.1.0/
+---
+Today I'd like to announce Homebrew 3.1.0. The most significant change since 3.0.0 os migrating our bottles (binary packages) to GitHub Packages.
```

Comment:
```suggestion
Today I'd like to announce Homebrew 3.1.0. The most significant change since 3.0.0 is migrating our bottles (binary packages) to GitHub Packages.
```

Reply:
I think this also ready better, but not a strong opinion, so feel free to ignore:

```suggestion
Today I'd like to announce Homebrew 3.1.0. The most significant change since 3.0.0 is the migration of our bottles (binary packages) to GitHub Packages.
```